samples/apps/copilot-chat-app/copilotchat/Models/AskResult.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+using System.Linq;
+
+namespace SemanticKernel.CopilotChat.Models;
+
+public class AskResult
```

Comment:
Same: re-usable

Reply:
Circular dependency.

Reply:
We don't necessarily need to use Ask for chat. Will be considered at the 2nd round of refactoring.